app/controllers/Api/V0/parks_controller.rb

```diff
@@ -1,9 +1,19 @@
 class Api::V0::ParksController < ApplicationController
   def index
-    render json: ParkSerializer.new(ParkFacade.get_parks)
+    data = ParkFacade.get_parks
+
+    if data.empty?
+      render json: {error: "No parks found"}, status: 404
+    else
+      render json: ParkSerializer.new(data)
+    end
   end
 
   def show
-    render json: ParkSerializer.new(ParkFacade.get_park(params[:id]))
+    if params[:id].length != 4
```

Comment:
does this id refer to the 4 letter park code or the 36 char "id" ?

Reply:
Talking to the FE guys it was a little more straightforward to go with the 4 letter park code